src/NorthwindWebsite/API/Controllers/ProductsController.cs

```diff
@@ -40,8 +38,8 @@ public async Task<ActionResult<Product>> GetById(int id)
     }
 
     [HttpPost]
-    [ProducesResponseType(StatusCodes.Status201Created)]
-    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(Product),StatusCodes.Status201Created)]
+    [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
```

Comment:
As far as I see you don't put anything inside of `BadRequest`, you can replace it with `typeof(void)`.
Fix it for other controller actions, please